package.json

```diff
@@ -30,6 +30,8 @@
     "xmlbuilder": "8.2.2"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
Also confirmed, we don't need to add these lines. Our imports already have them (probably via `webpack`).